src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = (
+  { person: { name, age, sex, isMarried, partnerName } },
+) => {
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  const getPartner = () => {
+    if (isMarried) {
+      return `${partnerName} is my ${partner}`;
+    }
+
+    return 'I am not married';
+  };
```

Comment:
You can do the same with a ternary operator as you do it for the `partner` variable.
```const partnerText = isMarried ? `${partnerName} is my ${partner}` : 'I am not married';```